_content/cannon/bodhicaryavatara2_santideva.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Bodhicaryāvatāra Chapter 2: Confession of Negativity"
+authors:
+  - santideva
+translator: "Adam Pearcey"
+external_url: "https://www.lotsawahouse.org/indian-masters/shantideva/bodhicharyavatara-2"
+drive_links:
+  - "https://drive.google.com/file/d/1S7Ku9ZkxAwRj7XlEJIWIyNcylQJPuCMI/view?usp=sharing"
+course: cosmology
+tags:
+  - thought
+  - bodhisatta
+  - confession
+  - offering
+year: 2007
+publisher: "Lotsawa House"
+pages: 8
+---
+
+A verse translation of chapter 2 from the Bodhisattvacaryāvatāra on “Confession of Negativity.”
```

Comment:
link to `the
[_Bodhisattvacaryāvatāra_](/content/canon/bodhisattvacaryavatara_santideva)`